samples/stock/demo/stock-css-design/demo.html

```diff
@@ -0,0 +1,18 @@
+
+
+<script src="https://code.highcharts.com/stock/highstock.js"></script>
+
+<script src="https://code.highcharts.com/stock/indicators/indicators-all.js"></script>
+<script src="https://code.highcharts.com/stock/modules/drag-panes.js"></script>
+
+<script src="https://code.highcharts.com/modules/annotations-advanced.js"></script>
+<script src="https://code.highcharts.com/modules/price-indicator.js"></script>
+<script src="https://code.highcharts.com/modules/full-screen.js"></script>
+
+<script src="https://code.highcharts.com/modules/stock-tools.js"></script>
+
+<script src="https://code.highcharts.com/stock/modules/heikinashi.js"></script>
+<script src="https://code.highcharts.com/stock/modules/hollowcandlestick.js"></script>
+<script src="https://code.highcharts.com/modules/accessibility.js"></script>
+
+<div id="container" class="chart"></div>
```

Comment:
On systems with light color scheme by default, the styles in demo.css are not sufficient, for example the Stock Tools button are all light. The fix is simply to set the `.highcharts-dark` class name on the container.

`<div id="container" class="chart highcharts-dark"></div>`